package.json

```diff
@@ -4,11 +4,10 @@
   "description": "Next.js & React app",
   "scripts": {
     "start": "npm run dev",
-    "dev": "next -p 3000",
+    "dev": "node server.js",
```

Comment:
Are we can specify port here for explicity?

Reply:
we can add something like that:
`export PORT=3000 && node server.js`
makes sense: that ensures `PORT` variable set for e.g. for Backend won't affect Frontend.